sa/_db/migrations/20150826133823_SCTReceipts.sql

```diff
@@ -0,0 +1,22 @@
+
+-- +goose Up
+-- SQL in section 'Up' is executed when this migration is applied
+
+CREATE TABLE `sctReceipts` (
+  `id` bigint(20) NOT NULL AUTO_INCREMENT,
+  `sctVersion` tinyint(3) NOT NULL,
+  `logID` varchar(255) NOT NULL,
+  `timestamp` bigint(20) NOT NULL,
+  `extensions` mediumblob,
+  `signature` mediumblob,
+  `certificateSerial` varchar(255) NOT NULL,
+  `LockCol` bigint(20) DEFAULT NULL,
+  PRIMARY KEY (`id`),
+  UNIQUE KEY `log_serial` (`logID`,`certificateSerial`)
```

Comment:
So this index that is created is only valid when you SELECT with both keys; since the above SELECT only has one, it's not going to be able to use the BTREE and will have to scan.